arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -688,7 +688,7 @@ Result RestReplicationHandler::testPermissions() {
               std::string dbName = _request->databaseName();
               DatabaseFeature& databaseFeature =
                   _vocbase.server().getFeature<DatabaseFeature>();
-              TRI_vocbase_t* vocbase = databaseFeature.lookupDatabase(dbName);
+              auto vocbase = databaseFeature.useDatabase(dbName);
```

Comment:
We need to hold it?

Reply:
We need it because some code further down calls `vocbase->lookupCollection(...)`.